Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,53 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Node Loss
+
+When a node is lost, you will see application pods on the node stuck in the `Terminating` state while another pod is rescheduled and is in the `ContainerCreating` state.
+
+To allow the application pod to start on another node, Kubernetes requires you to force delete the pod. After a timeout (8-10 minutes), your pod will be allowed to start on another node.
+
+### Force delete your application pod. After some timeout (8-10 minutes), your pod will be rescheduled
```

Comment:
No need for this line, right? It's already described on the previous line.
```suggestion
```

Reply:
I kept this and change previous line as this is a header which is separating two steps.

Reply:
so I removed ` Kubernetes requires you to force delete the pod. After a timeout (8-10 minutes), your pod will be allowed to start on another node.` from ```
To allow the application pod to start on another node, Kubernetes requires you to force delete the pod. After a timeout (8-10 minutes), your pod will be allowed to start on another node.``` I wanted to keep that as header below.
I wanted it as a header to look